specifications/tcp/MCtcp.tla

```diff
@@ -0,0 +1,24 @@
+---- MODULE MCtcp ----
+EXTENDS tcp, TLC
+
+CONSTANTS A, B
+peers == {A, B}
+Symmetry == Permutations(peers)
+
+NoRetransmission ==
+    \A p \in Peers :
+        \A i \in 1..Len(network[p]) - 1 :
+            network[p][i] # network[p][i + 1]
+
+SingleActiveOpen ==
```

Comment:
I assume you include actions ACTIVE_OPEN, PASSIVE_OPEN, and SEND in the overall spec in order to reflect the RFC but then you model check assuming that they do not happen, which explains the somewhat asymmetric initial condition?